packages/babel-helper-wrap-function/src/index.ts

```diff
@@ -87,26 +92,26 @@ function plainFunction(path: NodePath<any>, callId: any, noNewArrows: boolean) {
   }
 
   const built = callExpression(callId, [node]);
+  const params = node.params.reduce(
+    (acc, param) => {
+      acc.done = acc.done || isAssignmentPattern(param) || isRestElement(param);
+
+      if (!acc.done) {
+        acc.params.push(path.scope.generateUidIdentifier("x"));
+      }
+
+      return acc;
+    },
+    {
+      params: [],
+      done: false,
+    },
+  ).params;
```

Comment:
I know that you didn't write this code but just move it from a few lines below, however it's an example of "overusing reduce" and we can probably simplify it since we are touching these lines anyway:
```suggestion
  const params = [];
  for (const param of node.params) {
    if (isAssignmentPattern(param) || isRestElement(param)) break;
    params.push(path.scope.generateUidIdentifier("x"));
  }
```